ec_grasp_planner/src/handarm_parameters.py

```diff
@@ -66,71 +65,64 @@ class RBOHandP24WAM(RBOHand2):
-                                [0, 1, 0]))
+        # the maximum allowed force for pushing down
+        self['SurfaceGrasp']['object']['downward_force'] = 4
 
-        # second motion after grasp, in hand palm frame
-        self['surface_grasp']['object']['go_up_transform'] = tra.concatenate_matrices(tra.translation_matrix([-0.03, 0, -0.3]),
-                                                                            tra.rotation_matrix(math.radians(-20.),
-                                                                                                [0, 1, 0]))
+        # speed of approaching the object
+        self['SurfaceGrasp']['object']['down_speed'] = 0.05
 
-        # the maximum allowed force for pushing down
-        self['surface_grasp']['object']['downward_force'] = 4
+        # synergy type for soft hand closing
+        self['SurfaceGrasp']['object']['hand_closing_synergy'] = 1
 
-        #drop configuration - this is system specific!
-        self['surface_grasp']['object']['drop_off_config'] = np.array(
-            [-0.57148, 0.816213, -0.365673, 1.53765, 0.30308, 0.128965, 1.02467])
+        # time of soft hand closing
+        self['SurfaceGrasp']['object']['hand_closing_duration'] = 5
 
-        #synergy type for soft hand closing
-        self['surface_grasp']['object']['hand_closing_synergy'] = 1
+        # first motion after grasp, in hand palm frame
+        self['SurfaceGrasp']['object']['post_grasp_twist'] = np.array([0.0, 0.0, 0.0, 0.0, math.radians(-15.), 0])
 
-        #time of soft hand closing
-        self['surface_grasp']['object']['hand_closing_duration'] = 5
+        # duration of post grasp twist
+        self['SurfaceGrasp']['object']['post_grasp_rotation_duration'] = 2.
 
-        # time of soft hand closing
-        self['surface_grasp']['object']['down_speed'] = 0.35
-        self['surface_grasp']['object']['up_speed'] = 0.409
-        self['surface_grasp']['object']['go_down_velocity'] = np.array(
-            [0.125, 0.09])  # first value: rotational, second translational
-        self['surface_grasp']['object']['pre_grasp_velocity'] = np.array([0.125, 0.08])
+        # speed of lifting the object
+        self['SurfaceGrasp']['object']['up_speed'] = 0.05
+
+        # duration of lifting the object
+        self['SurfaceGrasp']['object']['lift_duration'] = 8
 
 
         #####################################################################################
         # below are parameters for wall grasp with P24 fingers (standard RBO hand)
         #####################################################################################
-        self['wall_grasp']['object']['hand_closing_duration'] = 1.0
-        self['wall_grasp']['object']['initial_goal'] = np.array(
-            [0.439999, 0.624437, -0.218715, 1.71695, -0.735594, 0.197093, -0.920799])
+        
+        self['WallGrasprasp']['object']['initial_goal'] = np.array([0.439999, 0.624437, -0.218715, 1.71695, -0.735594, 0.197093, -0.920799])
```

Comment:
'WallGrasprasp' is either a typo or a confusing name

Reply:
It was a typo. Fixed in https://github.com/SoMa-Project/ec_grasp_planner/pull/47/commits/e5e7630281a10fd60b9ca7a7dc7fc2a8cdd383ec